resources/views/portal/ninja2020/gateways/wepay/signup/index.blade.php

```diff
@@ -0,0 +1,14 @@
+@extends('portal.ninja2020.layout.clean')
+@section('meta_title', ctrans('texts.sign_up_with_wepay'))
+
+@section('body')
+
+    @livewire('wepay-signup', ['user_id' => $user_id, 'company_key' => $company_key])               
```

Comment:
If possible it'd be awesome if we can pass $user & $company directly to prevent double resolving.